src/api/app/controllers/webui/comments_controller.rb

```diff
@@ -14,7 +14,17 @@ def create
                flash.now[:error] = "Failed to create comment: #{comment.errors.full_messages.to_sentence}."
                :unprocessable_entity
              end
-    render(partial: 'webui/comment/comment_list', locals: { commentable: @commentable }, status: status)
+
+    if Flipper.enabled?(:request_show_redesign, User.session)
```

Comment:
This will break the "normal" comment flow...

Reply:
![Screenshot from 2022-08-09 16-45-35](https://user-images.githubusercontent.com/514785/183679710-048a02d1-a0ed-44c7-9555-1cb68c756e2e.png)

Reply:
Okay, Iet me see what's going on

Reply:
It's fixed now